embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/ndk/NdkService.kt

```diff
@@ -1,17 +1,11 @@
 package io.embrace.android.embracesdk.ndk
 
-import io.embrace.android.embracesdk.payload.NativeCrashData
-
-internal interface NdkService {
+internal interface NdkService : NativeCrashService {
     fun updateSessionId(newSessionId: String)
     fun onSessionPropertiesUpdate(properties: Map<String, String>)
     fun onUserInfoUpdate()
     fun getUnityCrashId(): String?
 
-    // TODO: remove this. Only for testing purposes.
```

Comment:
These methods are only invoked in tests, and the test, and the test is testing other test methods..... delete!